conftest.py

```diff
@@ -8,3 +9,12 @@ def global_setup_and_teardown():
   # setup a clean environment for each test
   with OpenpilotPrefix():
     yield
+
+
+def pytest_collection_modifyitems(config, items):
```

Comment:
I'd make this even more generic than "integration". it's more like we don't want to discover these tests, but only run them if they were specifically invoked

Reply:
perhaps "manual" is better? "tests that are only meant to be run manually, for example on-device in jenkins"

this would also be applicable to the simulation tests, some of the GPS tests that use the spoofer, etc